src/commonMain/kotlin/exchange.dydx.abacus/output/Asset.kt

```diff
@@ -112,3 +112,16 @@ data class Asset(
         }
     }
 }
+
+@JsExport
+@Serializable
+enum class AssetTags(val rawValue: String) {
```

Comment:
Where is this getting used?

Reply:
I was thinking that they could be exported for use purely on the client side

Reply:
the raw values are the coinmarketcap sector_tags that we want to support